jenkins/databricks/init_cuda11_runtime.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#
+# Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# The init script to install cuda11.0 toolkit
+# Will be automatically pushed into the dbfs:/databricks/init_scripts once it is updated.
+
+wget http://developer.download.nvidia.com/compute/cuda/11.0.2/local_installers/cuda_11.0.2_450.51.05_linux.run
+
+sudo sh cuda_11.0.2_450.51.05_linux.run --toolkit --silent
```

Comment:
so does this make the runtime show up in apt as well?   I assume based on the --toolkit parameter it only installed toolkit and cuda 11.0 runtime and the driver version status the same? (I haven't used this method before so just double checking).  Any difference between this and the instructions I sent as far as runtime?

Reply:
> so does this make the runtime show up in apt as well? 
Sorry, what does `show up in apt` mean?

Reply:
> I assume based on the --toolkit parameter it only installed toolkit and cuda 11.0 runtime and the driver version status the same? 
Correct

Reply:
> Any difference between this and the instructions I sent as far as runtime?

Should be the same,  Both `*.run[local]`  and `deb[local]`  download `toolkit + dirvers` tar to local, untar and install.

`*.run[local]`   CLI seems simple and clean.

Reply:
> > so does this make the runtime show up in apt as well?
> > Sorry, what does `show up in apt` mean? Do you mean when installing, are there any dialog showing up to let user select what to install(cuda toolkit/dirver/examples)? The pop up dialog is disabled by `--silence`